node/src/controller/intent.ts

```diff
@@ -0,0 +1,105 @@
+/**
+ * intent.ts
+ * Airwallex Payments Demo. Created by Shirly.Chen
+ * 
+ * This file describes that how to create a paymentIntent and retrieve a paymentIntent.
+ */
+import express, { Response } from "express";
+import axios from "axios";
+import { v4 as uuid } from "uuid";
+import config from "../config";
+
+const intentRouter = express.Router();
+
+// Create the PaymentIntent on the backend.
+intentRouter.get("/create", async (req, res) => {
```

Comment:
```suggestion
// Since this is a demo, to simplify the flow you can using a get call to create an intent, when doing integration you can change to post along with payload your shopper provided
intentRouter.get("/create", async (req, res) => {
```